sky/spot/constants.py

```diff
@@ -8,5 +8,8 @@
 
 SPOT_TASK_YAML_PREFIX = '~/.sky/spot_tasks'
 
-SPOT_WORKDIR_BUCKET_NAME = 'sky-spot-workdir-{username}-{hash}'
-SPOT_FILE_MOUNT_BUCKET_NAME = 'sky-spot-fm-{username}-{hash}'
+SPOT_WORKDIR_BUCKET_NAME = 'skypilot-spot-workdir-{username}-{id}'
```

Comment:
1. L11-14 have sky and skypilot. Should we be consistent?
2. Consider removing the "spot" word. When we add serverless I imagine these buckets can be reused.  Wdyt?